ccd-definition/FixedLists/JudgmentOnline/ConfirmReferToJudgeDefenceReceived.json

```diff
@@ -0,0 +1,8 @@
+[
```

Comment:
Isn't the "JO-nonprod" ending required in the file name?

Reply:
Yes! you are right!  Thank you Ignacio